core/src/main/java/com/linecorp/armeria/server/ParameterizedPathMapping.java

```diff
@@ -116,12 +125,20 @@
                 continue;
             }
 
+            if (isCaptureTheRestPathParam(token)) {
+                captureTheRestPathNames.add(paramName);
```

Comment:
Can `captureTheRestPathNames` contain more than one `paramName`?
If not, could we use a flag?
```java
final boolean isMultipleSegmentMatching = ...;
...
if (isMultipleSegmentMatching) {
    patternJoiner.add("(.*)");
} else {
    ...
}
```

Reply:
> Can captureTheRestPathNames contain more than one paramName?

No `CaptureTheRestPathParam` can include only 1 param in the path.
So, we can use the flag.
Ah, we can just use `isCaptureTheRestPathParam` method on L137 directly without defining `isCaptureTheRestPathParam` of `Set` like
```
if (isCaptureTheRestPathParam(token)) {
   patternJoiner.add("(.*)");
} else {
   patternJoiner.add("([^/]+)");
}
```